Makefile

```diff
@@ -23,14 +23,15 @@ IMAGE_DEV_ENV_BUILD ?= 0
 
 # Enable GO111MODULE=on explicitly, disable it with GO111MODULE=off when necessary.
 export GO111MODULE := on
-GOOS   := $(if $(GOOS),$(GOOS),"")
-GOARCH := $(if $(GOARCH),$(GOARCH),"")
-GOENV  := GO15VENDOREXPERIMENT="1" CGO_ENABLED=0 GOOS=$(GOOS) GOARCH=$(GOARCH)
-CGOENV := GO15VENDOREXPERIMENT="1" CGO_ENABLED=1 GOOS=$(GOOS) GOARCH=$(GOARCH)
-GO     := $(GOENV) go
-CGO    := $(CGOENV) go
-GOTEST := USE_EXISTING_CLUSTER=false NO_PROXY="${NO_PROXY},testhost" go test
-SHELL  := bash
+GOOS   	:= $(if $(GOOS),$(GOOS),"")
+GOARCH 	:= $(if $(GOARCH),$(GOARCH),"")
+GOPROXY := $(if $(GOPROXY),$(GOPROXY),"direct")
```

Comment:
Or `https://proxy.golang.org,direct`?

Sorry for wrong opinion before. I just prefer the real default value of `go`. It seems that the default value of `go` is `https://proxy.golang.org,direct`

Reply:
Or could we unset the `GOPROXY` env, if it's not set by the user?

Reply:
OK, we can just set `GOPROXY=""`, the go binary will use `GOPROXY="https://proxy.golang.org,direct"`.

Reply:
It seems `GOPROXY=""` will cause some errors in the go code.